static/app/views/settings/organizationMembers/inviteBanner.spec.tsx

```diff
@@ -0,0 +1,182 @@
+    });
+
+    render(
+      <InviteBanner
+        missingMembers={missingMembers}
+        onSendInvite={() => undefined}
+        organization={org}
+      />
+    );
+
+    await act(tick);
+
+    expect(
+      screen.queryByText('Bring your full GitHub team on board in Sentry')
+    ).not.toBeInTheDocument();
+  });
+
+  it('does not render banner if no missing members', async function () {
+    const org = TestStubs.Organization({
+      features: ['integrations-gh-invite'],
+    });
+
+    render(
+      <InviteBanner
+        missingMembers={noMissingMembers}
+        onSendInvite={() => undefined}
+        organization={org}
+      />
+    );
+
+    await act(tick);
+
+    expect(
+      screen.queryByText('Bring your full GitHub team on board in Sentry')
+    ).not.toBeInTheDocument();
+  });
+
+  it('does not render banner if lacking org:write', async function () {
+    const org = TestStubs.Organization({
+      features: ['integrations-gh-invite'],
+      access: [],
+    });
+
+    render(
+      <InviteBanner
+        missingMembers={noMissingMembers}
+        onSendInvite={() => undefined}
+        organization={org}
+      />
+    );
+
+    await act(tick);
+
+    expect(
+      screen.queryByText('Bring your full GitHub team on board in Sentry')
+    ).not.toBeInTheDocument();
+  });
+
+  it('renders banner if snoozed_ts days is longer than threshold', async function () {
+    const org = TestStubs.Organization({
+      features: ['integrations-gh-invite'],
+    });
+    const promptResponse = {
+      dismissed_ts: undefined,
+      snoozed_ts: moment
+        .utc()
+        .subtract(DEFAULT_SNOOZE_PROMPT_DAYS + 1, 'days')
+        .unix(),
+    };
+    MockApiClient.addMockResponse({
+      url: '/prompts-activity/',
+      method: 'GET',
+      body: {data: promptResponse},
+    });
+
+    render(
+      <InviteBanner
+        missingMembers={missingMembers}
+        onSendInvite={() => undefined}
+        organization={org}
+      />
+    );
+
+    await act(tick);
+
+    expect(
+      screen.getByText('Bring your full GitHub team on board in Sentry')
+    ).toBeInTheDocument();
+  });
+
+  it('does not render banner if snoozed_ts days is shorter than threshold', async function () {
+    const org = TestStubs.Organization({
+      features: ['integrations-gh-invite'],
+    });
+    const promptResponse = {
+      dismissed_ts: undefined,
+      snoozed_ts: moment
+        .utc()
+        .subtract(DEFAULT_SNOOZE_PROMPT_DAYS - 1, 'days')
+        .unix(),
+    };
+    MockApiClient.addMockResponse({
+      url: '/prompts-activity/',
+      method: 'GET',
+      body: {data: promptResponse},
+    });
+
+    render(
+      <InviteBanner
+        missingMembers={missingMembers}
+        onSendInvite={() => undefined}
+        organization={org}
+      />
+    );
+
+    await act(tick);
+
+    expect(
+      screen.queryByText('Bring your full GitHub team on board in Sentry')
+    ).not.toBeInTheDocument();
```

Comment:
this test can fail without actually checking anything. maybe add a check for the `prompts-activity` api to have been called